first-order-odes/separable.md

```diff
@@ -22,6 +22,44 @@ y &= c e^\frac{x^2}{2}
 Note that here, we are being careful to denote the redefinition of the
 integration constant ($c = e^{c_0}$). This detail may be glossed over at times.
 
+Solve the following:
+
+(a)
+
+$$
```

Comment:
```suggestion
```

$$ is used to wrap equations that are on the same line as normal text. It is not needed here.